Rakefile

```diff
@@ -6,19 +6,21 @@ require "date"
 task default: :formula_and_analytics
 
 desc "Dump macOS formulae data"
-task :formulae, [:os] do |task, args|
-  args.with_defaults(:os => "mac")
+task :formulae, [:os,:tap] do |task, args|
+  args.with_defaults(:os => "mac", :tap => "")
 
-  ENV["HOMEBREW_FORCE_HOMEBREW_ON_LINUX"] = "1" if args.os == "mac"
+  ENV["HOMEBREW_FORCE_HOMEBREW_ON_LINUX"] = "1" if args[:os] == "mac"
```

Comment:
Had to switch `args.tap` to `args[:tap]` because .tap conflicts with the `Object.tap` ruby method. To keep things consistent within this block, the .os invocations were converted as well.